src/formatDate.js

```diff
@@ -50,7 +50,36 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const mainDate = date.split(fromFormat[fromFormat.length - 1]);
```

Comment:
Consider using destructuring to extract the separator from 'fromFormat'. For example: const [separator] = fromFormat.slice(-1);